app/serializers/service_provider_serializer.rb

```diff
@@ -17,7 +17,7 @@ class ServiceProviderSerializer < ActiveModel::Serializer
   )
 
   def agency
-    object.name
+    object.friendly_name
```

Comment:
shouldn't this be `agency` instead -- in which case, no explicit method definition required for `agency` or `friendly_name` since they are now column names.